server/app/views/components/Accordion.java

```diff
@@ -57,7 +57,7 @@ public DivTag getContainer() {
             .withClasses(
                 ReferenceClasses.ACCORDION_BUTTON,
                 "transition-all",
-                "duration-300",
+                "duration-100",
```

Comment:
I decreased the spinning caret transition time on the accordion to make tests less flaky and because I saw no visual improvement with a slower caret spin.